app/views/users/show.html.haml

```diff
@@ -0,0 +1,36 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
+          %h2 #{@user.full_name}'s video collections (#{@user.queue_items.count})
+          = link_to 'Follow', relationships_path(followed_user_id: @user.id), method: :post, class: 'button btn btn-default' if current_user.can_follow?(@user)
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queue_items.each do |queue_item|
+              %tr
+                %td
+                  = link_to queue_item.video_title, video_path(queue_item.video)
+                %td
+                  = link_to queue_item.category_name, category_path(queue_item.video.category)
```

Comment:
You can do `queue_item.video.category` instead of `category_path(queue_item.video.category)`.